src/novmm/loader/linux_setup.go

```diff
@@ -69,75 +71,75 @@ static inline void set_header(
 import "C"
 
 import (
-    "novmm/machine"
-    "novmm/platform"
-    "unsafe"
+	"novmm/machine"
+	"novmm/platform"
+	"unsafe"
 )
 
 func SetupLinuxBootParams(
-    model *machine.Model,
```

Comment:
On Wed, Jan 7, 2015 at 3:02 PM, Pekka Enberg <notifications@github.com>
wrote:

> In src/novmm/loader/linux_setup.go
> <https://github.com/google/novm/pull/1#discussion-diff-22615558>:
>
> >  )
> >
> >  func SetupLinuxBootParams(
> > -    model *machine.Model,
>
> Looks like you reformatted the whole file with gofmt inserting tabs
> instead of four spaces for indent.
>
Yeah, I can't get "gofmt -l=true -w=true -tabs=false -tabwidth=4
src/novmm/*"  on Fedora 21 to work. "-tabs" is not valid.

[tjyang@fedora21 novm]$ gofmt -l=true -w=true -tabs=false -tabwidth=4
src/novmm/*
flag provided but not defined: -tabs
usage: gofmt [flags] [path ...]
  -cpuprofile="": write cpu profile to this file
  -d=false: display diffs instead of rewriting files
  -e=false: report all errors (not just the first 10 on different lines)
  -l=false: list files whose formatting differs from gofmt's
  -r="": rewrite rule (e.g., 'a[b:len(a)] -> a[b:]')
  -s=false: simplify code
  -w=false: write result to (source) file instead of stdout
[tjyang@fedora21 novm]$


tj

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/novm/pull/1/files#r22615558>.
>



-- 
T.J. Yang